infra/terraform/modules/ses_domain/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_ses_domain_identity" "domain" {
+  domain = "${var.domain}"
+}
+
+# SES Verification: TXT Record
+resource "aws_route53_record" "amazonses_verification_record" {
+  zone_id = "${var.aws_route53_zone_id}"
+
+  name = "_amazonses.${aws_ses_domain_identity.domain.id}"
+
+  type = "TXT"
+  ttl  = "1800"
+
+  records = [
+    "${aws_ses_domain_identity.domain.verification_token}",
+  ]
+}
+
+resource "aws_ses_domain_identity_verification" "amazonses_verification" {
+  domain = "${aws_ses_domain_identity.domain.id}"
+
+  depends_on = ["aws_route53_record.amazonses_verification_record"]
+}
+
+# SES Verification: DKIM
+resource "aws_ses_domain_dkim" "domain_verification" {
+  domain = "${aws_ses_domain_identity.domain.domain}"
+}
+
+resource "aws_route53_record" "domain_amazonses_dkim_verification_record" {
+  count   = 3
```

Comment:
I'd say this should be `count = ${length(aws_ses_domain_dkim.domain_verification.dkim_tokens)}`, in case AWS change the number of DNS records used for DKIM verification.

Reply:
Sorted: https://github.com/ministryofjustice/analytics-platform-ops/pull/162/commits/db400cc48f530a7ef9607f923f14a9925dc8c8d1